subsys/net/lib/lwm2m/lwm2m_message_handling.c

```diff
@@ -172,6 +172,45 @@ static void free_block_ctx(struct lwm2m_block_context *ctx)
 	ctx->tkl = 0U;
 }
 
+static int init_message_pending(struct lwm2m_message *msg)
```

Comment:
It makes it harder to review when the actual changes introduced are mixed up with moving code around (like introducing helper function here). This should be done in a separate commits.